.github/CONTRIBUTING.md

```diff
@@ -1,17 +1,17 @@
 ## Reporting Security Vulnerabilities
 
-If you think that you have found a security vulnerability, please report it at [https://huntr.dev](https://huntr.dev) with as much detail as possible. The Core Team will review the vulnerability and if found applicable, will assign a CVE ID and issue a fix. The vulnerability will be disclosed once the patch has been included into a release. 
+If you think that you have found a security vulnerability, please report it at [https://huntr.dev](https://huntr.dev) with as much detail as possible. The Core Team will review the vulnerability and if found applicable, will assign a CVE ID and issue a fix. The vulnerability will be disclosed once the patch has been included into a release.
```

Comment:
The security vulnerabilities have a different process now as well. Do you think we can include this fix in your PR?
```suggestion
If you think that you have found a security vulnerability, please report it at [https://www.mautic.org/mautic-security-team/how-to-report-a-security-issue](https://www.mautic.org/mautic-security-team/how-to-report-a-security-issue) with as much detail as possible. The Core Team will review the vulnerability and if found applicable, will assign a CVE ID and issue a fix. The vulnerability will be disclosed once the patch has been included into a release.
```